Options.lua

```diff
@@ -225,7 +225,7 @@ function WarpDeplete:InitOptions()
 			},
 			general = group(L["General"], false, {
 				lineBreak(),
-				toggle(L["Insert keystone automatically"], "insertKeystoneAutomatically", "RenderLayout"),
+				toggle(L["Insert Keystone Automatically"], "insertKeystoneAutomatically", "RenderLayout"),
```

Comment:
I'm generally not a fan of "drive-by fixes" like this, since that makes rolling changes back harder.

If you want to include this in this PR anyways, please go through all the translations files and update they translation keys for other languages. The strings here are used as keys for localization and this will break other translations otherwise.